lua/pixelui/core/cl_images.lua

```diff
@@ -20,15 +20,14 @@ local queue = {}
 
 local useProxy = false
 
-file.CreateDir("pixel")
 file.CreateDir(PIXEL.DownloadPath)
 
 local function processQueue()
     if queue[1] then
         local url, matSettings, callback = unpack(queue[1])
 
-        local filePath = PIXEL.DownloadPath .. "/" .. url
-        url = "https://" .. url
+        local filePath = PIXEL.DownloadPath .. url
+        url = "http://" .. url
```

Comment:
You also cannot assume HTTP is supported, this is something users should be able to specify.